soc/hps_lattice_nx.py

```diff
@@ -77,7 +77,7 @@ def initval_parameters(contents, width):
 
 
 class NXLRAM(Module):
-    def __init__(self, width=32, size=128*kB, dual_port=False, init=[]):
+    def __init__(self, width=32, size=128*kB, dual_port=False, init=[], cen={"a": 0b1, "b": 0b1}):
```

Comment:
I was thinking more about this. I think it would be cleaner and more consistent to make the A and B clock enable signals be attributes on this module, rather than passed in at construction. We already have `b_addrs` and `b_douts` attributes, we could add `a_clkens` and `b_clkens` the same way. It would also mean a bit less passing down this cen parameter through various functions.

What do you think?

Reply:
Sure, I'm not a big fan of my approach, I don't consider it intuitive and that passing parameters doesn't look good but I didn't think about that too much. Your idea seems to be better. I'll apply it, thanks.